src/SerilogWeb.Classic/Classic/ApplicationLifecycleModule.cs

```diff
@@ -45,7 +45,7 @@ public static ILogger Logger
         {
             get
             {
-                return _logger ?? Log.Logger;
+                return _logger ?? Log.Logger.ForContext<ApplicationLifecycleModule>();
```

Comment:
Perhaps some parens here would make it consistent between default and user-specified logger?

```csharp
               return (_logger ?? Log.Logger).ForContext<ApplicationLifecycleModule>();
```